script.js

```diff
@@ -1,53 +1,58 @@
-let bookList = [];
-let book;
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+const books = [];
 let title;
 let author;
 
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
-}
+class UI {
+  static bookDisplay(title, author) {
+    const bookList = document.getElementById('book-list');
 
-function displayBook(title, author) {
-  const div = document.createElement('div');
-  div.innerHTML = `
-  <p>${title}</p>
-  <p>${author}</p>
-  <button onclick = "remove('${title}', '${author}', this)" class="delete">Remove Book</button>`;
-  const list = document.getElementById('list');
-  list.appendChild(div);
-}
+    const row = document.createElement('tr');
+
+    row.innerHTML = `
+        <td>${title}</td>
+        <td>${author}</td>
+        <td><button class="delete">Remove Book</button></td>
```

Comment:
- [ ] **OPTIONAL**
It'd be better if you validated the form to prevent adding/spamming empty books.